.github/workflows/goreleaser.yml

```
@@ -8,23 +8,23 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Set env
         run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
       - name: Use Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v4
```

actionlint (1.7.11)

[error] the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)